specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,222 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` **MUST** be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
+
+
+## `X-Payment` header payload
```

Comment:
In my opinion, we must include a unique reference to the requested payment in the transaction, resolving the payment. Example: if I'm buying a book from the online store, then the payment is related to a specific purchase associated with a payment ID, which should be included in the payment note field.

Reply:
On the Whitepaper they explain that's part of the payment requirements response when you hit 402, it's part of the 402 protocol itself and not something we need to spec. I assume when we reply to 402 with the payment params, we include the payment ID. It's actually only mentioned once in the whitepaper and not part of the struct defined in the README. 

It's something we can clarify during the PR with coinbase and go from there